hr_timesheet_sheet/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_timesheet_sheet_user,hr_timesheet.sheet,model_hr_timesheet_sheet,base.group_user,1,0,0,0
```

Comment:
An employee (user belonging to group `base.group_user`) should have full access to the sheet (`1,1,1,1`), like it was in V10.
Could you check?
This way the line (`access_hr_timesheet_sheet_manager`) would be useless and could be removed.

Reply:
In v10, there are three lines of security

Reply:
yes I know ;)